kore/src/Kore/Step.hs

```diff
@@ -153,6 +224,7 @@ priorityAllStrategy
     => [rewrite]
     -> Strategy (Prim rewrite)
 priorityAllStrategy rewrites =
+    -- TODO: should this be Strategy.all?
```

Comment:
Neither `all` nor `first` is really correct. The rules should be grouped by priority and applied in parallel or sequence in the transition rule, just the same as is done in `Kore.Reachability.Claim`.